06_fusion_snapshots/01_compute_envs.md

```diff
@@ -0,0 +1,169 @@
+
+#### 2. Fusion V2 Compute Environment
+
+If we inspect the contents of [`./compute-envs/aws_fusion_ondemand.yml`](./compute-envs/aws_fusion_ondemand.yml) as an example, we can see the overall structure is as follows:
+
+```yaml
+compute-envs:
+  - type: aws-batch
+    config-mode: forge
+    name: "${COMPUTE_ENV_PREFIX}_fusion_ondemand"
+    workspace: "$ORGANIZATION_NAME/$WORKSPACE_NAME"
+    credentials: "$AWS_CREDENTIALS"
+    region: "$AWS_REGION"
+    work-dir: "$AWS_WORK_DIR"
+    wave: True
+    fusion-v2: True
+    fast-storage: True
+    no-ebs-auto-scale: True
+    provisioning-model: "SPOT"
+    instance-types: "c6id,m6id,r6id"
+    max-cpus: 1000
+    allow-buckets: "$AWS_COMPUTE_ENV_ALLOWED_BUCKETS"
+    labels: storage=fusionv2,project=benchmarking"
+    wait: "AVAILABLE"
+    overwrite: False
+```
+<details>
+<summary>Click to expand: YAML format explanation</summary>
+
+The top-level block `compute-envs` mirrors the `tw compute-envs` command. The `type` and `config-mode` options are seqerakit specific. The nested options in the YAML correspond to options available for the Seqera Platform CLI command. For example, running `tw compute-envs add aws-batch forge --help` shows options like `--name`, `--workspace`, `--credentials`, etc., which are provided to the `tw compute-envs` command via this YAML definition.
+
+</details>
+
+#### 3. Fusion Snapshots Compute Environment
+
+Fusion snapshots is a new feature in Fusion that allows you to snapshot and restore your machine when a spot interruption occurs. If we inspect the contents of [`./compute-envs/aws_fusion_snapshots.yml`](./compute-envs/aws_fusion_snapshots.yml) as an example, we can see the overall structure is as follows:
+
+```YAML
+compute-envs:
+  - type: aws-batch
+    config-mode: forge
+    name: "${COMPUTE_ENV_PREFIX}_fusion_snapshots"
+    workspace: "$ORGANIZATION_NAME/$WORKSPACE_NAME"
+    credentials: "$AWS_CREDENTIALS"
+    region: "$AWS_REGION"
+    work-dir: "$AWS_WORK_DIR"
+    wave: True
+    fusion-v2: True
+    fast-storage: True
+    no-ebs-auto-scale: True
+    provisioning-model: "SPOT"
+    instance-types: "c6id.4xlarge,c6id.8xlarge,r6id.2xlarge,m6id.4xlarge,c6id.12xlarge,r6id.4xlarge,m6id.8xlarge"
+    max-cpus: 1000
+    allow-buckets: "$AWS_COMPUTE_ENV_ALLOWED_BUCKETS"
+    labels: storage=fusionv2,project=benchmarking"
+    wait: "AVAILABLE"
+    overwrite: False
+```
+
+You should note it is very similar to the Fusion V2 compute environment, but with the following differences:
+
+- `provisioning-model` is set to `SPOT` to enable the use of spot instances.
+- `instance-types` are set to a very restrictive set of types that have sufficient memory and bandwidth to snapshot the machine within the time limit imposed by AWS during a spot reclamation event.
+
+#### Pre-configured Options in the YAML
+
+We've pre-configured several options to optimize your Fusion snapshots compute environment:
+
+| Option | Value | Purpose |
```

Comment:
maybe swap Purpose and Value column. The instance types don't wrap and it is hard to read on the rendered version